lib/linguist/languages.yml

```diff
@@ -4325,6 +4325,7 @@ Shell:
   - bashrc
   - cshrc
   - gradlew
+  - mvnw
```

Comment:
filenames should be alphabetically sorted, should be added after "man" and before "profile"

Reply:
How many of these `mvnw` lack shebangs? Any portable shell-script is guaranteed to start with `#!/bin/sh` as per long-standing convention.

If there's no risk of a `mvnw` file lacking the hashbang, then you should omit this entry from `languages.yml` altogether. It's unrelated to your classification, and I believe the cause of the problem is to do with vendored-file detection, not languages.

Reply:
> How many of these `mvnw` lack shebangs?

All the `mvnw` files should start with `#!/bin/sh` as it has been there since the Maven Wrapper project has been started 5 years ago (see git blame [here](https://github.com/takari/maven-wrapper/blame/master/mvnw#L1)).

> If there's no risk of a mvnw file lacking the hashbang, then you should omit this entry from `languages.yml` altogether

ack, I'll push a commit removing `mvnw` from `languages.yml` as there is no risk of having `mvnw` files not starting by a shebang `#!/bin/sh`.

Thanks for your help @Alhadis !